scripts/wrapper/withdraw.js

```diff
@@ -64,18 +65,26 @@ module.exports = function (web3, artifacts) {
       const tokenBracketPairs = []
       for (const tokenData of tokenDataList)
         for (const bracketAddress of brackets) tokenBracketPairs.push([bracketAddress, tokenData])
+
       const maxWithdrawableAmounts = await Promise.all(
         tokenBracketPairs.map(([bracketAddress, tokenData]) => amountFunction(bracketAddress, tokenData, exchange))
       )
       withdrawals = []
-      maxWithdrawableAmounts.forEach((amount, index) => {
-        if (amount !== "0")
+      for (const [index, amount] of maxWithdrawableAmounts.entries()) {
+        const token = tokenBracketPairs[index][1]
+        const bracketAddress = tokenBracketPairs[index][0]
+        const usdValue = await amountUSDValue(amount, token, globalPriceStorage)
+        if (usdValue.gte(ONE)) {
           withdrawals.push({
-            bracketAddress: tokenBracketPairs[index][0],
-            tokenAddress: tokenBracketPairs[index][1].address,
-            amount: amount,
+            bracketAddress,
+            tokenAddress: token.address,
+            amount,
+            usdValue,
```

Comment:
Why do we store this value? Will it ever be used?

Reply:
This was something I meant to delete although I thought it might be used in this way at some point.